pynav/lib/models.py

```diff
@@ -214,6 +362,90 @@ def covariance(self, x: VectorState) -> np.ndarray:
         return self._R
 
 
+class RelativeLandmark(MeasurementModel):
+    def __init__(
+        self,
+        landmark_position: np.ndarray,
+        R: np.ndarray,
+    ):
+        self._landmark_position = landmark_position
+        self._R = R
+
+    def evaluate(self, x: MatrixLieGroupState) -> np.ndarray:
+        """Evaluates the measurement model of a landmark from a given pose."""
+        r_zw_a = x.position.reshape((-1, 1))
+        C_ab = x.attitude
+        r_pw_a = self._landmark_position.reshape((-1, 1))
+        return C_ab.T @ (r_pw_a - r_zw_a)
+
+    def jacobian():
+        pass
+
+    def covariance(self, x):
+        return self._R
+
+
+class InvariantRelativeLandmark(MeasurementModel):
```

Comment:
The default `InvariantInnovation` would work, but only if the user has already specified the Jacobians of the regular measurement model. In this case I wanted to directly specify the Jacobians of the invariant innovation, so I changed `InvariantMeasurement` to optionally accept a measurement model, otherwise it defaults to the `_InvariantInnovation`. I'm not sure this is the best solution in the case that the user directly wants to specify Jacobians of the innovation, but we can definitely think about other ways to do it

Reply:
I see. I think its fine to have a format where the invariant innovation is define directly, but at the same time i also see the use in a non-invariant version's jacobian, in case someone wants to implement a non-invariant filter